packages/common/package.json

```diff
@@ -62,6 +62,7 @@
         "hash.js": "^1.1.7",
         "iso-3166-1": "^2.1.1",
         "json-bigint": "^1.0.0",
+        "leb128": "^0.0.5",
```

Comment:
Is size a concern here? `leb128` depends on `bn.js` for big integers instead of using the native `BigInt` which I expect would increase the size.